src/shell/main.cpp

```diff
@@ -101,7 +101,7 @@ static command_executor commands[] = {
         "get the node status for this cluster",
         "[-d|--detailed] [-j|--json] [-r|--resolve_ip] [-u|--resource_usage] "
         "[-o|--output file_name] [-s|--status all|alive|unalive] [-q|--qps] "
-        "[-t|--sample_interval_ms num]",
+        "[-i|--sample_interval_ms num]",
```

Comment:
It should be mentioned in "Behavior changes" in the PR description for this PR.

Reply:
OK, the options `sample_interval_ms` was introduced recently and never released before.